lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -249,6 +233,19 @@ sub _resolve_reference {
 sub __errors__ {
     my $self   = shift;
     my @errors = $self->SUPER::__errors__;
+
+    for my $model (qw(wgs_model exome_model)) {
+        if ($self->$model) {
+            unless ($self->$model->isa('Genome::Model::SomaticVariation') || $self->$model->isa('Genome::Model::SomaticValidation')) {
```

Comment:
Would `$self->$model->last_succeeded_build->isa('Genome::Model::Build::SomaticInterface')` do the trick? Maybe with an additional check that `last_succeeded_build` exists? Or does the inheritance hierarchy not support this?